client/src/pages/AssessmentPage/AssessmentPage.jsx

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import {
+  Typography,
+  CssBaseline,
+  Button,
+  TextField,
+  Box,
+  FormControl,
+  RadioGroup,
+  FormControlLabel,
+  Radio
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: "flex",
+    flexWrap: "wrap"
+  },
+  question: {
+    width: "75vw",
+    maxWidth: "300px"
+  }
+}));
+
+//TODO: Import the question from the database and update the state values
+
+function AssessmentPage(props) {
+  const classes = useStyles();
+  const [value, setValue] = React.useState("");
+  const handleChange = event => {
+    setValue(event.target.value);
+  };
+  const handleSubmit = () => {
+    //TODO: store answers
+  };
+  const questions = [
+    {
+      question: "This is the first question",
+      type: "mc",
+      answers: ["Choice 0", "Choice 1", "Choice 2", "Choice 3", "Choice 4"],
+      number: "1"
+    },
+    {
+      question: "This is the second question",
+      type: "sa",
+      answers: [],
+      number: "2"
+    },
+    {
+      question: "This is the third question",
+      type: "mc",
+      answers: ["Choice 0", "Choice 1", "Choice 2", "Choice 3", "Choice 4"],
+      number: "3"
+    },
+    {
+      question: "This is the fourth question",
+      type: "mc",
+      answers: ["Choice 0", "Choice 1", "Choice 2", "Choice 3", "Choice 4"],
+      number: "4"
+    },
+    {
+      question: "This is the sixth question",
+      type: "mc",
+      answers: ["Choice 0", "Choice 1", "Choice 2", "Choice 3", "Choice 4"],
+      number: "6"
+    }
+  ];
+  var states = {};
```

Comment:
(nitpick) In js, use _let_ or _const_ when declaring variables as opposed to _var_ (modern standards don't use js)